core/templates/dev/head/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -76,6 +79,20 @@ angular.module('oppia').directive('storyNodeEditor', [
               }
             }
           };
+          var _isExpIdValid = function(expId) {
+            // If user has not yet entered an exploration id
+            if (expId === null) {
+              return true;
```

Comment:
I would suggest let's not call this function if `expId` is `null`. As of now it looks like null is a valid expId.

Reply:
Done